app/app.js

```diff
@@ -25,7 +27,7 @@ import configureStore from './store';
 
 // Observe loading of Open Sans (to remove open sans, remove the <link> tag in
 // the index.html file and this observer)
-import styles from 'containers/App/styles.css';
+import styles from './containers/App/styles.css';
```

Comment:
> I don't think that it's because of webpack.

That's definitely because of webpack, we use the `resolve` option to specify which directories weback should search in. That's why in your components you can do

```JS
// components/Something/index.js

import SomethingElse from 'containers/SomethingElse';
```

and that'll work. (notice the non-relative path) I think the problem here is in our eslint-import-plugin setup, though I think we had that working before…